src/elements/content-sharing/SharingNotification.js

```diff
@@ -199,16 +202,20 @@ function SharingNotification({
         api,
         itemID,
         itemType,
-        ownerEmail,
-        currentUserID === ownerID,
-        null,
+        data => convertCollabsResponse(data, ownerEmail, currentUserID === ownerID),
         () => handleError(contentSharingMessages.collaboratorsLoadingError),
     );
-    setCollaboratorsList(collaboratorsList);
+    if (collaboratorsList && !collabsExist) {
+        setCollaboratorsList(collaboratorsList);
```

Comment:
Typically, I haven't seen our `successCallback`s return data (L205) -- could we call `setCollaboratorsList(convertCollabsResponse(...))` here?